logging/slog.go

```diff
@@ -0,0 +1,40 @@
+package logging
+
+import (
+	"errors"
+	"log/slog"
+	"os"
+)
+
+var logger *slog.Logger
+
+// LevelHandler determines which type of Handler should be used for the logger
+type LevelHandler int
```

Comment:
This is not a level handle it is a Handler Option

change the type name to `Handler`

Reply:
That's correct!
I've changed it to `HandlerType`